phpBB/phpbb/storage/exception/not_implemented.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\storage\exception;
+
+class not_implemented extends \phpbb\exception\runtime_exception
```

Comment:
couldn't it extends `phpbb\storage\exception` to avoid code duplication ? (and semantically it makes sense too)